src/vunnel/workspace.py

```diff
@@ -243,17 +244,17 @@ def overlay_existing(self, source: str, move: bool = False) -> None:
                 os.makedirs(os.path.dirname(dst), exist_ok=True)
 
                 if move:
-                    os.rename(src, dst)
+                    shutil.move(src, dst)
```

Comment:
Actually, we might need the same change here, and it's just not failing a test.

Reply:
Is `overlay_existing` actually used anywhere?  I can't find any invocations of it.  Maybe we can just remove it?

Reply:
I don't see it used anywhere. I think it was added by mistake in https://github.com/anchore/vunnel/pull/520, since that PR went through a lot of iterations.

I've pushed a commit removing the function.